dim/doc/api.rst

```diff
@@ -280,6 +280,11 @@ Pool Functions
    Remove the vlan from *pool* and all its subnets.
 
 
+.. function:: ippool_set_layer3domain(pool, layer3domain)
+
+   Set layer3domain for *pool* and all its subnets.
```

Comment:
Would this be a good helptext?

```
`ippool_set_layer3domain` moves *pool* from its origin layer3domain to *layer3domain*. Subnets of the *pool*, the contained entries and delegations are also moved to *layer3domain*.
Reverse entries are stored in *views* named after the *layer3domain*. All reverse entries are being deleted from the old *view* and then created in the new *view*. If the *view* does not exist, it will be created.
Please remember to add eventually newly created views to their appropriate zone-profile.
Forward entries will be fixed to keep their reference to the reverse record.

This operation can take a long time to succeed depending on the number of existing subnets, IPs, forward DNS entries and reverse DNS entries. In this time the pool and its entries can't be modified.
```

Reply:
Yes, thanks.
Having a more detailed description is much better, especially in this case.
I've updated the description according to your suggestion.